ansible/configs/sap-hana-auto/software.yml

```diff
@@ -0,0 +1,48 @@
+---
+- name: Step 00xxxxx software
+  hosts: localhost
+  gather_facts: False
+  become: false
+  tasks:
+    - debug:
+        msg: "Software tasks Started"
+
+- name: Install and configure NFS Server
+  hosts: bastion
+  become: True
+  gather_facts: True
+  roles:
+    # Install and deloy NFS Server
+    - { role: "host-ocp-nfs", when: install_nfs }
+  tasks:
+    - name: Ensure Ansible is installed
+      yum:
+        name: ansible
+        state: latest
+
+- name: Deploy Ansible Tower
+  hosts: towers
+  gather_facts: False
+  become: yes
+  tasks:
+    # This is a bit hacky but we are facing some issues with Ansible, RHEL8 and python for some
+    # modules and this workaround solved this particular issue
+    - name: Ensure Python3 package is installed and alternatives for python updated
+      shell: >
+        yum install -y python3 && alternatives --set python /usr/bin/python3
```

Comment:
can you split this shell task in 2 tasks, 1 to install the package using `yum` or `package` module, and another one to set alternatives ?

Reply:
I tried that already and setting alternatives in a different task after python3 has been installed fails as there is no default python in RHEL 8. Will give it another try with the new image you guys pushed and update here